indexer/locality_object.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "base/buffer_vector.hpp"
+#include "base/osm_id.hpp"
+
+#include <cstdint>
+#include <vector>
+
+namespace indexer
+{
+class LocalityObject
+{
+public:
+  static osm::Id FromStoredId(uint64_t storedId) { return osm::Id(storedId >> 2 | storedId << 62); }
```

Comment:
Please add an explanation to why this encoding is chosen.

Reply:
done